vulture_os/toolkit/api_parser/gatewatcher_alerts/gatewatcher_alerts.py

```diff
@@ -77,6 +80,18 @@ def execute_query(self, url, params=None, timeout=20):
             timeout=timeout,
             verify=self.api_parser_custom_certificate or self.api_parser_verify_ssl
         )
+        if response.status_code == 404:
+            logger.warning(f"[{__parser__}]:execute_query: 404 error at Gatewatcher API Call, content : {response.content}", extra={'frontend': str(self.frontend)})
+            # The cause of the 404 error can be that the former endpoint us used
+            url = url.replace("/api/", "/api/v1/")
+            response = self.session.get(
+                url,
+                params=params,
+                proxies=self.proxies,
+                timeout=timeout,
+                verify=self.api_parser_custom_certificate or self.api_parser_verify_ssl
+            )
+
```

Comment:
Not sure handling the switch to the legacy API path in the code is a good idea, shouldn't you let the user select the version of the API they want to use instead?